UPGRADE-3.x.md

```diff
@@ -32,6 +32,22 @@ extend the corresponding classes in `SonataAdminBundle\Action\`.
 If you need to style headers prefer to use CSS classes and not in the html DOM.
 In this case please use `header_class` option.
 
+## Deprecated return type of SimplePager::getResults
+When calling `SimplePager::getResults()` on non-empty result which extends `$maxPerPage` value, you will get `ArrayCollection` instead of `array`
```

Comment:
Ok this is why I got confused: "you will" is future tense, which implies that you now get an `array`, and will get an `ArrayCollection` on next major, which is the opposite of what will actually happen.